### DISCUSSION TITLE
最便宜iPhone掰了？分析師爆蘋果已取消SE 4開發計畫

### DISCUSSION THREAD

**Discussion by ABCDEFGHIJKLMNO12345678** [1101-GS4719-C/Python-Community-GS4719-C#1083] [General]
從2016年開始 蘋果所推出的SE系列成為許多想用蘋果但有無法付擔如此高的價格的第二首選 但最近供佈由于5G晶片發展的不順可能最新一代的SE會停產 對於許多果粉來說無非是個壞消息

https://tw.news.yahoo.com/%E6%9C%80%E4%BE%BF%E5%AE%9Ciphone%E6%8E%B0%E4%BA%86-%E5%88%86%E6%9E%90%E5%B8%AB%E7%88%86%E8%98%8B%E6%9E%9C%E5%B7%B2%E5%8F%96%E6%B6%88se-4%E9%96%8B%E7%99%BC%E8%A8%88%E7%95%AB-084159131.html

**Comment by Valentinaa0214** [comment]
我自己現在就是使用se2，
因為很喜歡小手機拿在手上很輕便的這種感覺，
這應該也是很多人選擇se系列的原因，
看到它不繼續開發有點難過。

**Comment by Twenty0522** [comment]
我目前是拿蘋果高階的手機，但回去看以前的手機，拿起來很輕巧，設計也很適合手比較小的女生，希望蘋果可以針對不同的客群保留各種特色的產品。

**Comment by ghost** [comment]
我不是蘋果用戶，但以我的認知蘋果的產品走的都是精品路線，講究用戶體驗，所以在研發上成本自然高，再加上人們對蘋果的預期心理，產品就是要有驚艷的感覺，所以當平民機一出，自然就會...。

**Comment by yuxinlin93** [comment]
這個價錢跟其他蘋果手機的價錢比起來便宜蠻多的，感覺不錯，雖然沒辦法用大螢幕爽爽看影片，但小小的尺寸看起來真的很可愛，而且對女生來說比較好拿，比較不會出現在床上滑手機被砸臉的慘案，沒能做出來實在有些可惜
